gguf-py/gguf/constants.py

```diff
@@ -844,6 +852,9 @@ class MODEL_TENSOR(IntEnum):
     MODEL_TENSOR.POSNET_ATTN_K:             "posnet.{bid}.attn_k",
     MODEL_TENSOR.POSNET_ATTN_V:             "posnet.{bid}.attn_v",
     MODEL_TENSOR.POSNET_ATTN_OUT:           "posnet.{bid}.attn_output",
+    MODEL_TENSOR.SHORTCONV_CONV:            "blk.{bid}.shortconv.conv",
+    MODEL_TENSOR.SHORTCONV_INPROJ:          "blk.{bid}.shortconv.in_proj",
+    MODEL_TENSOR.SHORTCONV_OUTPROJ:         "blk.{bid}.shortconv.out_proj",
```

Comment:
Using `shortconv` prefix would be more consistent with the existing naming pattern:

```suggestion
    MODEL_TENSOR.SHORTCONV_CONV:            "shortconv.{bid}.shortconv.conv",
    MODEL_TENSOR.SHORTCONV_INPROJ:          "shortconv.{bid}.shortconv.in_proj",
    MODEL_TENSOR.SHORTCONV_OUTPROJ:         "shortconv.{bid}.shortconv.out_proj",
```

Reply:
I think `blk.*.shortconv` is probably fine, because the HF tensor name is `model.layers.*.shortconv`, meaning it's part of an existing layer (same as attn and FFN blocks)

<img width="672" height="278" alt="image" src="https://github.com/user-attachments/assets/55d99431-7949-474e-8ce8-dd758ff3f8b8" />

With the `shortconv.{bid}` prefix, the list of tensors may look a bit strange, as it's duplicated the list of `blk`

```
blk.0.*
shortconv.0.*
blk.1.*
shortconv.1.*
blk.2.*
shortconv.2.*
...
```